massa-node/src/main.rs

```diff
@@ -601,6 +607,8 @@ struct Args {
     /// Wallet password
     #[structopt(short = "p", long = "pwd")]
     password: Option<String>,
+    #[structopt(short = "b", long = "boot")]
+    boot: Option<String>,
```

Comment:
Passing an argument to use a config file is complex for user. The most easy way for the user to do it it's like our config.toml. We create a file by default in your case whitelist: [] and then the user can edit it.
Sadly this doesn't match the requirements of the issue that ask this list to be live editable. I think a way to do it live executable is to use the API.